language/move-lang/tests/functional/validator_set/add_validator.move

```diff
@@ -0,0 +1,53 @@
+// Add simple validator to LibraSystem's validator set.
+
+//! account: bob, 1000000, 0, validator
+
+//! sender: bob
+script {
+use 0x0::LibraSystem;
+use 0x0::ValidatorConfig;
+fun main() {
+    // test bob is a validator
+    0x0::Transaction::assert(ValidatorConfig::is_valid({{bob}}) == true, 98);
+    0x0::Transaction::assert(LibraSystem::is_validator({{bob}}) == true, 98);
+}
+}
+
+// check: EXECUTED
+
+//! new-transaction
+//! sender: association
+script {
+use 0x0::LBR;
+use 0x0::LibraAccount;
+fun main() {
+    LibraAccount::create_validator_account<LBR::T>(0xAA, x"00000000000000000000000000000000");
+}
+}
+
+// check: EXECUTED
+
+// TODO(valerini): enable the following test once the sender format is supported
```

Comment:
should we fix this now? or is it still work in progress?

Reply:
I hope it will get fixed very soon - there is a discussion in slack #dev-platform. But it's still work in progress (by Victor). I can add a temporary function to the module to re-enable this test, but I'd rather wait for it to be fixed... Since especially the validators' creation functionality is being tested with creating validators for these functional tests themselves.